iot-hub/Samples/device/IoTHubDeviceSamples.sln

```diff
@@ -13,6 +13,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MethodSample", "MethodSampl
 EndProject

```

Comment:
Please add a "run" statement to execute the sample in https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/build.ps1#L105
As pointed above, that would be complicated as your sample requires several items that are not available in the SDK (such as a proper edge configured device).